serde-generate/src/generate.rs

```diff
@@ -67,9 +69,13 @@ struct Options {
     /// Optional package name (Python) or module path (Go) where to find Serde runtime dependencies.
     #[structopt(long)]
     serde_package_name: Option<String>,
+
+    /// Translate enums without variant data (c-style enums) into their equivalent in the target language.
+    #[structopt(long)]
+    c_style_enums: bool,
```

Comment:
FYI I may add a verb here later, like `use_c_style_enums`

Reply:
That does seem more consistent with `--with-runtimes`